src/mongoose_api_common.erl

```diff
@@ -327,6 +327,8 @@ error_response(ErrorType, Reason, Req, State) ->
                     L when is_list(L) -> list_to_binary(L);
                     Other -> list_to_binary(io_lib:format("~p", [Other]))
                 end,
+    ?LOG_ERROR(#{what => rest_common_error,
+                 error_type => ErrorType, reason => Reason, cowboy_req => Req}),
```

Comment:
Now the only thing that concerns me is the fact that we log here `cowboy_req`. Doesn't that might contain private data? We might leak here.